src/network/serverpackethandler.cpp

```diff
@@ -526,7 +530,9 @@ void Server::handleCommand_Init_Legacy(NetworkPacket* pkt)
 		actionstream << "Server: " << playername << " tried to join, but there"
 				<< " are already max_users="
 				<< g_settings->getU16("max_users") << " players." << std::endl;
-		DenyAccess_Legacy(pkt->getPeerId(), L"Too many users.");
+		DenyAccess_Legacy(pkt->getPeerId(), L"There are currently too many users on this"
```

Comment:
This is different from https://github.com/minetest/minetest/pull/2786/files#diff-d6c71fb6c2c5e2ad979f39a0422f7786R951